start_esp/nginx-auto.conf.template

```diff
@@ -48,10 +48,7 @@ http {
 % endif
 % endfor
 
-  set_real_ip_from  0.0.0.0/0;
-  set_real_ip_from  0::/0;
-  real_ip_header    X-Forwarded-For;
-  real_ip_recursive on;
```

Comment:
ngx_http_realip_module doesn't allow duplication. 
https://github.com/nginx/nginx/blob/master/src/http/modules/ngx_http_realip_module.c#L425

And it doesn't support multiple header in the setting either.
https://github.com/nginx/nginx/blob/master/src/http/modules/ngx_http_realip_module.c#L214

We can add an command line argument to choose which header.